packages/material-ui-lab/src/DatePicker/DatePicker.tsx

```diff
@@ -124,6 +133,7 @@ DatePicker.propTypes /* remove-proptypes */ = {
   components: PropTypes.shape({
     LeftArrowButton: PropTypes.elementType,
     LeftArrowIcon: PropTypes.elementType,
+    openPickerIcon: PropTypes.node,
```

Comment:
It's a component, not a React node

```suggestion
    openPickerIcon: PropTypes. elementType,
```